configuration/pih/htmlforms/hiv/section-pmtct-history.xml

```diff
@@ -169,32 +168,41 @@
                          conceptId="PIH:REFERRAL OR TRANSFER" style="radio"
                          answerConceptIds="PIH:11521,PIH:6965"
                          answerCodes="pihcore.referral,pihcore.transfer"
-                         answerSeparator="" toggle="transfer-referral-out-details"/>
+                         answerSeparator="">
+                        <controls>
+                            <when value="PIH:11521" thenDisplay="#transfer-referral-details"/>
+                            <when value="PIH:6965" thenDisplay="#transfer-referral-details"/>
+                        </controls>
+                    </obs>
                 </p>
-                <div id="transfer-referral-out-details">
+                <div id="transfer-referral-details">
                     <p class="side-by-side">
-                        <obs id="transfer-out-spots"
+                        <obs id="transfer-spots"
                              conceptId="PIH:REFERRED FROM ANOTHER SITE" style="radio"
                              answerConceptIds="PIH:8855,PIH:8856"
                              answerCodes="pihcore.internal,pihcore.external"
                              answerSeparator="">
                             <controls>
-                                <when value="PIH:8855" thenDisplay="#internal-out"/>
-                                <when value="PIH:8856" thenDisplay="#external-out"/>
+                                <when value="PIH:8855" thenDisplay="#internal-transfer"/>
+                                <when value="PIH:8856" thenDisplay="#external-transfer"/>
                             </controls>
                         </obs>
                     </p>
 
-                    <p id="internal-out">
+                    <p id="internal-transfer">
                         <label>
-                            <uimessage code="pihcore.internalLocation"/>
+                            <uimessage code="pihcore.internalTransferLocation"/>
                         </label>
                         <obs conceptId="PIH:8621" style="location" answerLocationTags="Program Location"/>
+                        <includeIf velocityTest="$patient.gender == 'F' ">
```

Comment:
Since this is the PMTCT form, I don't think you need to check gender=F @mogoodrich .  I doesn't break anything tho.

Reply:
Yeah, i cut-and-pasted from the HIV history, didn't seem necessary but also seemed like it wouldn't hurt.